flox/aggregations.py

```diff
@@ -470,6 +470,19 @@ def _pick_second(*x):
 # median = Aggregation("median", chunk=None, combine=None, fill_value=None)
 # nanmedian = Aggregation("nanmedian", chunk=None, combine=None, fill_value=None)
 
+# Cumulatives:
+cumsum_ = Aggregation(
```

Comment:
Let's rename `reduction_type` to `kind` and set it to `cumulative` here.

Reply:
Done